packages/lesswrong/lib/collections/votes/views.ts

```diff
@@ -65,7 +66,7 @@ Votes.addView("userVotes", function ({collectionNames,}, _, context?: ResolverCo
       isUnvote: {$ne: true},
       // only include neutral votes that have extended vote data
       $or: {
-        voteType: {$ne: "neutral"},
+        voteType: "neutral",
```

Comment:
This renders the comment above false, also I don't know what the desired behavior is? I think this change is probably wrong